ykbh/src/lib.rs

```diff
@@ -0,0 +1,310 @@
+#![feature(test)]
+extern crate test;
+
+use ykpack::{self, Local, Terminator, Statement, IPlace, Constant, ConstantInt, UnsignedInt};
+use yktrace::sir::SIR;
+use std::convert::TryInto;
+
+const ZST: Object = Object { data: Vec::new() };
+
+#[derive(Clone, Debug)]
+/// A data object, storing bytes.
+pub struct Object {
+    data: Vec<u8>
```

Comment:
I think you said you were going to try and use something more efficient than many vectors (for another PR, right?).

We could use one shared (and growing, to account for call frames) vector, and an "Object" is a slice into that?

Reply:
Is this the same ideas as in 2) of the PR description?

Reply:
It's subtly different. I was proposing one vector for all frames. I don't think that'll work though. See my other comment later about growing/reallocating such a vector.

One vec per-frame would work though. We statically know the size of the frame.